src/index-node.ts

```diff
@@ -28,9 +28,21 @@ export class SiweClient extends SiweImpl {
 
   async _extractCookies(headers?: Headers): Promise<void> {
     const cookieRecord: Record<string, string> = {}
-    const headerSetCookies = (headers as any).raw()[
-      'set-cookie'
-    ] as Array<string>
+    // previously
```

Comment:
```suggestion
```
it seems like this was left by mistake. maybe you were going to write what was done here previously, but I'd prefer if we left that kind of change logging to the Git history.